protocol/0012-POSR-position_resolution.md

```diff
@@ -22,147 +22,45 @@ Any trader that has insufficient collateral to cover their margin liability is r
+  - if party's margin account balance is now above or equal to their latest maintenance margin level then such party is no longer marked as "distressed".
 
-No traders are removed from the distressed trader list.
+- If party's is marked as "distressed" at this stage then:
 
-#### STEP 2
+  - their open volume get's added to network's open volume for that market,
+  - party's open volume in that market gets set to 0,
+  - the full amount of party's margin account balance gets transferred to market's insurance pool.
 
-`NetOutstandingLiability = 5 - 4 + 2 = 3`
+This concludes the position resolution from party's perspective.
 
-#### STEP 3
+#### Managing network's position
 
-```json
-LiquiditySourcingOrder: {
-  type: 'market',
-  direction: 'sell',
-  size: 3
-}
+Whilst network has a non-zero position in a given market it's treated as any other party with market's insurance in that market acting as its margin account and with and exception that margin search, margin release or liquidation are never attempted on a network party.
 
-LiquiditySourcingTrade1: {
-  buyer: Trader4,
-  seller: Network,
-  size: 2,
-  price: 120,
-  type: 'liquidity-sourcing'
-}
+Whenever the network party has a non-zero position it attempts to unload it using an [immediate or cancel](./0014-ORDT-order_types.md) limit order. If the the network has a `long` position it will submit a `sell` order. If it has a short position it will submit a buy order. The size of that order is chosen according to the liquidation strategy which forms a part of the marke's configuration. The strategy can be updated at any point whilst market is active with a market change [governance vote](./0028-GOVE-governance.md#2-change-market-parameters).
 
-LiquiditySourcingTrade2: {
-  buyer: Trader5,
-  seller: Network,
-  size: 1,
-  price: 100,
-  type: 'liquidity-sourcing'
+The liquidation strategy consists of:
 
-}
+- `disposal time step` (mandatory, e.g. `n=10s`): network attempts to unload its position in a given market every time it goes out of auction and then every `n` seconds as long as market is not in auction mode and while the network's position is not equal to `0`,
+- `disposal fraction` (e.g. `0.1`): fraction of network's current open volume that it will try to reduce in a single disposal attempt,
+- `full disposal size` (e.g. `20`): once net absolute value of network's open volume is at or below that value, the network will attempt to dispose the remaining amount in one go,
+- `max fraction of book side within liquidity bounds consumed` (e.g. `m=0.05`): once the network chooses the size of its order (`s_candidate`) the effective size will be calcualted as `s_effective=min(m*N, s_candidate)`, where `N` is the sum of volume (on the side of the book with which the network's order will be matching) that falls within the range implied by the `market.liquidity.priceRange` [parameter](./0044-LIME-lp_mechanics.md#market-parameters),
```

Comment:
minimum valid value is `0` - which means that the network party can never dispose of anything, maximum valid value is `1.0`, default value is `0.05`. Note that when doing `m x N` we should be rounding up.